docs/gtm-interchain.md

```diff
@@ -0,0 +1,73 @@
+
+&quot;_Never doubt that a small group of thoughtful, committed, citizens can change the world. Indeed, it is the only thing that ever has.&quot; - Margaret Mead_
+
+**What&#39;s Needed to Successfully Go to Market?**
+
+1. **Project management**
+
+In sum, project management will focus on surfacing issues, dependencies, relative roadmaps, and coordinating across all stakeholders. Questions that will be considered throughout each phase of the Gaia roadmap:
+
+- What is missing from an integration points perspective with end users?
+
+- Which relationships are missing?
+- What&#39;s the highest priority? Least priority?
+
+2. **Market validation**
+
+The user is paramount. For Gaia and IBC, we see the users being:
+
+  - Atom holders
+  - Existing zones
+  - New zones
+  - Validators on hub and zones
+  - Nodes on hub and zones
+  - Users of hub and zones
+
+We will also want to be tracking outreach to the relevant users per phase to ensure that Hub utility (and IBC where appropriate) are being validated. Spotlights on experiments will continue to validate the utility of the Hub and IBC (i.e. bridge to Ethereum, AMM, etc). We will also be conducting continous outbound outreach to look for partners that are aligned with the interchain mission.
+
+3. **Communications**
+
+Communications should focus on the features of IBC and the Hub that demonstrate the most utility for users. Stories should answer questions such as:
+
+- How does IBC relate to ATOM and the Hub?
+- How does IBC derive its value from the Hub?
+- Why should you build your blockchain with IBC?
+- How does IBC help blockchains become sovereign?
+
+4. **Technical Support**
+
+We will need technical support for users depending on the phase that is being rolled out. A phase rollout is not complete without proper technical support.
+
+Further, we will need technical support that goes beyond the different phase rollouts. Importantly, to create an IBC connected blockchain developers will need the following things:
+
+- A value proposition
+
+  - A State Machine:
+    - Built on Cosmos SDK
+    - ABCI
+    - Cosmos-rs.. etc
+  - Security:
+    - Cross chain Validation
+    - Validator set formation
+  - Community
+    - Discoverability Opportunities
+  - Connectivity
+  - Visibility
+    - Wallet, Block Explorer, etc
+
+5. **METRICS: How do we know when IBC is succeeding? How do we know when Hub is providing value to an interconnected ecosystem of sovereign blockchains?**
+
+The go-to-market team should be tracking the extent to which IBC is succeeding or failing in terms of key metrics. This will require continuous experimentation and coordination between entities. An example of a metric that best represents IBC go to market would be # of distinct IBC connections that transport x amount of value. To get to these metrics we need the developer resources and proven business models that encourage developers to use this technology.
```

Comment:
```suggestion
The go-to-market team should be tracking the extent to which IBC is succeeding or failing in terms of key metrics. This will require continuous experimentation and coordination between entities. An example of a metric that best represents IBC go to market would be # of distinct IBC connections that transport x amount of value. To get to these metrics we need the developer resources and proven business models that encourage developers to use this technology.

The Cosmos-SDK exposes some [existing metrics for IBC](https://docs.cosmos.network/master/core/telemetry.html#supported-metrics).
```